scripts/krakenuniq

```diff
@@ -230,7 +231,7 @@ if ($paired) {
 my $cmd = $use_exact_counting? $CLASSIFY_EXACT : $CLASSIFY;
 print STDERR "$cmd @flags\n";
 if (defined $report_file) {
-    my $mycmd = qx/ps -o args= $$/;
+    my $mycmd = join(' ', $0, @cmdline);
     open(my $RF, ">", $report_file) or die "Could not open report file for writing";
     print $RF "# KrakenUniq v#####=VERSION=##### DATE:$date DB:@db_prefix DB_SIZE:$db_size WD:$wd\n# CL:$mycmd\n";
```

Comment:
I just noted an unintended side-effect of the suggestion in this PR: the report file no longer includes an empty line on line 3. If this is wanted to ensure compatibility with older versions it should probably be added in again. 

```suggestion
    print $RF "# KrakenUniq v#####=VERSION=##### DATE:$date DB:@db_prefix DB_SIZE:$db_size WD:$wd\n# CL:$mycmd\n\n";
```